charts/trino/README.md

```diff
@@ -63,6 +63,8 @@ The following table lists the configurable parameters of the Trino chart and the
 | `coordinator.resources` |  | `{}` |
 | `coordinator.livenessProbe` |  | `{}` |
 | `coordinator.readinessProbe` |  | `{}` |
+| `coordinator.lifecycle` |  | `{}` |
+| `coordinator.terminationGracePeriodSeconds` |  | `"30"` |
```

Comment:
Nit: remove double quotes

Reply:
Done

Reply:
the `CI/CD / docs (pull_request)`  check is failing because of this change

```bash
-| `coordinator.terminationGracePeriodSeconds` |  | `30` |
+| `coordinator.terminationGracePeriodSeconds` |  | `"30"` |
```

I'll rollback this change

Reply:
It's because I placed a string on `charts/trino/values.yaml`